src/itemviewformaction.cpp

```diff
@@ -696,11 +696,11 @@ void ItemViewFormAction::register_format_styles()
 {
 	std::string attrstr = rxman.get_attrs_stfl_string("article", false);
 	attrstr.append(
-		"@style_b_normal[color_bold]:attr=bold "
-		"@style_u_normal[color_underline]:attr=underline ");
```

Comment:
Was it correct to remove the attributes here? I don't see them ported to the new code.

Reply:
I think so.
These are handled in a part of ColorManager which is not touched by this PR:
https://github.com/newsboat/newsboat/blob/0c29d63085bc2d9a020930c43dafe06af60d69e7/src/colormanager.cpp#L170-L185